### PR TITLE
CDI Extension

### PR DESCRIPTION
**Pull request by fh-ms**
We will bring Otavio's CDI extension (currently in [own repo](https://github.com/microstream-one/microstream-cdi-extension)), to our main repo. The extension module itself is located in integrations/cdi, the examples in examples of course.
Builds and tests are running fine on my side, so there should be no problem.
I had to update all license headers to the EPL, please check again if the builder missed any files.

### CODE REVIEW

File: examples/cdi-javase/src/main/java/one/microstream/examples/cdi/javase/App.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+import javax.enterprise.inject.se.SeContainer;
+import javax.enterprise.inject.se.SeContainerInitializer;
+
+
+public class App
+{
+	
+	public static void main(final String[] args)
+	{
+		try(SeContainer container = SeContainerInitializer.newInstance().initialize()) {
```
Comment (zdenek-jonas) [inline examples/cdi-javase/src/main/java/one/microstream/examples/cdi/javase/App.java]: format?

File: pom.xml [hunk 2]
```diff
@@ -85,6 +85,26 @@
 				<artifactId>slf4j-api</artifactId>
 				<version>1.7.32</version>
 			</dependency>
+			<dependency>
+				<groupId>org.junit.jupiter</groupId>
+				<artifactId>junit-jupiter-engine</artifactId>
+				<version>5.8.2</version>
```
Comment (zdenek-jonas) [inline pom.xml:91]: scope test?

File: examples/helidon-mp/src/main/java/one/microstream/examples/helidonmp/HelloController.java [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+ */
+
+import javax.inject.Singleton;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+
+
+/**
+ *
+ */
```
Comment (zdenek-jonas) [inline examples/helidon-mp/src/main/java/one/microstream/examples/helidonmp/HelloController.java]: Emtpy javadoc

**Review by hg-ms — APPROVED** [review]
Didn't find missing licenses in java files

**Review by zdenek-jonas — APPROVED** [review]

File: examples/helidon-mp/src/main/java/one/microstream/examples/helidonmp/HelloController.java (hunk 3 above)
Comment (fh-ms) [reply examples/helidon-mp/src/main/java/one/microstream/examples/helidonmp/HelloController.java]: Removed empty javadocs

**Pull request merged**